<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>223</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NSimSun</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Numéros VIP</string>
  </property>
  <widget class="QLabel" name="lbl_titre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Numéros VIP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_num">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NSimSun</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_verif">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NSimSun</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vérifier / Proposer</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_res">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NSimSun</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
